next/components/forms/widget-components/RadioButton/Radio.tsx

```diff
@@ -66,10 +66,10 @@ const Radio = ({
 
   return (
     <div className="w-full">
-      <label htmlFor={rest.value} className={containerStyle}>
+      <label htmlFor={rest.value.toString()} className={containerStyle}>
```

Comment:
I would guess that value can end up being something like `undefined`, in which case `.toString()` would throw - please handle